wolfcrypt/src/integer.c

```diff
@@ -3477,23 +3480,25 @@ int fast_s_mp_mul_digs (mp_int * a, mp_int * b, mp_int * c, int digs)
       int      iy;
       mp_digit *tmpx, *tmpy;
 
-      /* get offsets into the two bignums */
-      ty = MIN(b->used-1, ix);
-      tx = ix - ty;
+      if ((a->used > 0) && (b->used > 0)) {
```

Comment:
Should there be an `else { error }` ?

Reply:
nope, it's not an error for them to be empty, and the code actually breaks if it errors there.  I know -- I tried.